.github/workflows/rebase-pr.yml

```diff
@@ -0,0 +1,20 @@
+name: Rebase a pr
+
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  rebase_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@master
+      - name: Rebase PR
+        if: |
+          github.event_name == 'pull_request' &&
+          github.event.action == 'labeled' &&
```

Comment:
Shouldn't need these first two since it will only be triggered on a labeled pull request from above.